tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -3752,6 +3771,12 @@ void SKEditorConsumer::handleSemanticAnnotation(unsigned Offset,
   SemanticAnnotations.add(Kind, Offset, Length, isSystem);
 }
 
+void SKEditorConsumer::handleDeclaration(unsigned Offset, unsigned Length,
+                                         UIdent Kind, StringRef USR) {
+  assert(Kind.isValid());
+  Declarations.add(Kind, Offset, Length, USR);
```

Comment:
Could check if declarations is enabled here?

Reply:
Oh, yes, sorry. Got carried away with copying the boilerplate there I suppose. Thanks!